cmd/create.go

```diff
@@ -94,8 +95,11 @@ func buildInstance(options *options.Options) (*computepb.Instance, error) {
 				},
 			},
 		},
+		Tags: buildInstanceTags(options),
 		NetworkInterfaces: []*computepb.NetworkInterface{
 			{
+				Network:    normalizeNetworkID(options),
```

Comment:
Thanks it seems good and useful :)

Just a question: If we do not specify a networkid or a subnetid, does this work with effectively passing `Network: nil`?
Because the normalize functions return nil when string is empty

Reply:
Thanks! That's fine, I was just curious because other providers would error out for this stuff
Then this is fine by me :)